subprojects/docs/src/docs/userguide/publishingIvy.adoc

```diff
@@ -56,12 +56,12 @@ Applying the “`ivy-publish`” plugin does the following:
 
 [NOTE]
 ====
-If you are not familiar with project artifacts and configurations, you should read <<artifact_management>>, which introduces these concepts. This chapter also describes “publishing artifacts” using a different mechanism than what is described in this chapter. The publishing functionality described here will eventually supersede that functionality.
+If you are not familiar with project artifacts and configurations, you should read <<artifact_management>>, which introduces these concepts. That chapter also describes publishing artifacts using a different mechanism than what is described in this chapter. The publishing functionality described here will eventually supersede that functionality.
```

Comment:
Not part of your change, but I prefer "a different mechanism that _the one_ described in this chapter".